concent_api/core/migrations/0009_auto_20180305_1119.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please squash all commits with migrations into single one, along with changes in models.